r/tests/testthat/test-parquet.R

```diff
@@ -322,3 +340,13 @@ test_that("ParquetFileWrite chunk_size calculation doesn't have integer overflow
   # but our max_chunks is respected
   expect_equal(calculate_chunk_size(101, 1, 25, 2), 51)
 })
+
+# Wait on ARROW-13735 for this
```

Comment:
I'd like us to be able to validate we can read all the test parquet files after this. (I think parquet support is the main motivation for MapType support; not sure it's much use otherwise.) Waiting on #11855, which solves a bug affecting one of these.